heron/instance/src/java/com/twitter/heron/instance/bolt/BoltInstance.java

```diff
@@ -232,7 +235,7 @@ public void readTuplesAndExecute(Communicator<Message> inQueue) {
       if (msg instanceof CheckpointManager.InitiateStatefulCheckpoint) {
         String checkpointId =
             ((CheckpointManager.InitiateStatefulCheckpoint) msg).getCheckpointId();
-        LOG.info("Start checkpoint for: " + checkpointId);
+        LOG.info("Persisting state for checkpoint: " + checkpointId);
```

Comment:
Do we need a `LOG.info(..)` here?

Reply:
This log message is indicating the instance is doing checkpoint, we should keep it. But I moved this `LOG.info()` into `persistState` method for better organization